tasks/maketree.js

```diff
@@ -0,0 +1,57 @@
+const { task } = require('hardhat/config');
```

Comment:
Why is this being added?

Reply:
the import of the function was missing on line 4